scripts/run.sh

```diff
@@ -124,6 +127,24 @@ while getopts ":i:d:m:p:jstvnrh" opt ; do
         v)
             PYTHON_COMMAND=${VISUALIZER_COMMAND}
             ;;
+        x)
+            # First check if .Xauthority exists
+            export DISPLAY=":10.0"
+            FILE=~/.Xauthority
+            if [ -f "$FILE" ]; then
+              echo "$FILE exists, able to render."
+            else
+              echo "$FILE doesn't exist, create it before running Docker with -x arg."
+              exit 126
+            fi
+            XSOCK=/tmp/.X11-unix
+            XAUTH=/tmp/.docker.xauth
+            touch $XAUTH
+            xauth nlist $DISPLAY | sed -e 's/^..../ffff/' | xauth -f $XAUTH nmerge -
```

Comment:
I'm getting this error now:
```
xauth:  timeout in locking authority file /tmp/.docker.xauth
```

would this be better if the temp file was created per user?
```suggestion
            XAUTH=/tmp/.docker.xauth.$USER
            touch $XAUTH
            xauth nlist $DISPLAY | sed -e 's/^..../ffff/' | xauth -f $XAUTH nmerge -
```

Reply:
Checking the error you get, agreed with your suggestion, I already added it.